README.md

```diff
@@ -193,6 +193,7 @@ constructorio.tracker.trackConversion('dogs', {
 | --- | --- |
 | `name` | string |
 | `customer_id` | string |
+| `variation_id` | string |
```

Comment:
Can we update to play nicely with https://github.com/Constructor-io/constructorio-client-javascript/pull/43?

Reply:
Updated, hopefully what i pushed up works fine when everything gets merged into master